pkg/registry/rbac/rest/storage_rbac.go

```diff
@@ -334,3 +342,51 @@ func primeAggregatedClusterRoles(clusterRolesToAggregate map[string]string, clus
 
 	return nil
 }
+
+// primeSplitClusterRoleBindings ensures the existence of target ClusterRoleBindings
+// by copying Subjects, Annotations, and Labels from the specified source
+// ClusterRoleBinding, if present.
+func primeSplitClusterRoleBindings(clusterRoleBindingToSplit map[string]rbacapiv1.ClusterRoleBinding, clusterRoleBindingClient rbacv1client.ClusterRoleBindingsGetter) error {
+	for clusterRoleBindingName, clusterRoleBindingToCreate := range clusterRoleBindingToSplit {
+		// If source ClusterRoleBinding does not exist, do nothing.
+		existingRoleBinding, err := clusterRoleBindingClient.ClusterRoleBindings().Get(clusterRoleBindingName, metav1.GetOptions{})
+		if apierrors.IsNotFound(err) {
+			return nil
+		}
+		if err != nil {
+			return err
+		}
+
+		// If the target ClusterRoleBinding already exists, do nothing.
+		_, err = clusterRoleBindingClient.ClusterRoleBindings().Get(clusterRoleBindingToCreate.Name, metav1.GetOptions{})
+		if err == nil {
+			return nil
+		}
+		if !apierrors.IsNotFound(err) {
+			return err
+		}
+
+		// If the source exists, but the target does not,
+		// copy the subjects, labels, and annotations from the former to create the latter.
```

Comment:
the subfields of `existingRoleBinding` can be used directly. can simplify to:

```go
newCRB := clusterRoleBindingToCreate.DeepCopy()
newCRB.Subjects = existingRoleBinding.Subjects
newCRB.Labels = existingRoleBinding.Labels
newCRB.Annotations = existingRoleBinding.Annotations
if _, err := clusterRoleBindingClient.ClusterRoleBindings().Create(newCRB) ...
```

Reply:
done